install.md

```diff
@@ -1,5 +1,36 @@
 # libpod Installation Instructions
 
+## Installing packaged versions of Podman
+
+#### Arch Linux
+
+https://aur.archlinux.org/packages/libpod/
+
+#### Fedora, Centos, OpenSUSE, SUSE:
+
+sudo yum -y install podman
+
+#### RHEL7
+
+Enable the extras repo.
+sudo yum -y install podman
+
+#### RHEL8 Beta
+
+sudo dnf module enable -y container-tools:1.0
+sudo dnf module install -y container-tools:1.0
+sudo yum -y install podman
+
+#### Ubuntu and friends
+...
+
+#### Portage
```

Comment:
Portage is (one of) the packging tool of gentoo, so this section should probably be merged into the next one